clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -934,7 +934,7 @@ void tools::addLTOOptions(const ToolChain &ToolChain, const ArgList &Args,
     std::optional<StringRef> OptVal =
         llvm::StringSwitch<std::optional<StringRef>>(ArgVecLib->getValue())
             .Case("Accelerate", "Accelerate")
-            .Case("LIBMVEC", "LIBMVEC-X86")
+            .Case("libmvec", "LIBMVEC-X86")
```

Comment:
Does the normalized value need changing to match this? Currently it's "LIBMVEC" all uppercase: 
https://github.com/llvm/llvm-project/blob/ee3610efb511ec0a8d5ebd2016ce23b4b4344cd1/clang/include/clang/Driver/Options.td#L3430-L3433

Reply:
I think that's fine because it is an enum value within llvm::driver::VectorLibrary (see llvm/include/llvm/Frontend/Driver/CodeGenOptions.h).

Reply:
Oh right I see now :+1:  I assumed this StringSwitch was correct, but it looks like `-fveclib=libmvec` didn't work with LTO before (only LIBMVEC uppercase).